docs/integrations/emr/networking.md

```diff
@@ -0,0 +1,99 @@
+# Networking
+
+In order for Spark to communicate with the Hopsworks Feature Store from EMR, networking needs to be set up correctly. This includes deploying the Hopsworks Feature Store to either the same VPC or enable VPC peering between the VPC of the EMR cluster and the Hopsworks Feature Store.
+
+## Step 1: Ensuring network connectivity
+
+The Spark DataFrames API needs to be able to connect directly to the IP on which the Feature Store is listening.
+This means that if you deploy the Feature Store on AWS you will either need to deploy the Feature Store in the same VPC as your EMR
+cluster or to set up [VPC Peering](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html) between your EMR VPC and the Feature Store VPC.
+
+**Option 1: Deploying the Feature Store in the EMR VPC**
+
+When deploying the Hopsworks Feature Store, select the EMR *VPC* and *Availability Zone* as the VPC and Availability Zone of your Feature Store.
+Identify your EMR VPC in the Summary of your EMR cluster:
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/emr/emr_vpc_0.png">
+      <img src="../../../assets/images/emr/emr_vpc_0.png" alt="Identifying the EMR VPC">
+    </a>
+    <figcaption>Identifying the EMR VPC</figcaption>
+  </figure>
+</p>
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/emr/emr_vpc_1.png">
+      <img src="../../../assets/images/emr/emr_vpc_1.png" alt="Identifying the EMR VPC">
+    </a>
+    <figcaption>Identifying the EMR VPC</figcaption>
+  </figure>
+</p>
+
+!!! info "Hopsworks installer"
+    If you are performing an installation using the [Hopsworks installer script](https://hopsworks.readthedocs.io/en/stable/getting_started/installation_guide/platforms/hopsworks-installer.html), ensure that the machines you are going to install Hopsworks on are configured with the respective VPC.
```

Comment:
What does "respective VPC" mean?
This?
"the virtual machines you install Hopsworks on are deployed in the EMR VPC."

Reply:
I don't really know, it's a copy and paste from the Databricks doc. I'll put what you suggest.